src/constants.js

```diff
@@ -1,5 +1,4 @@
-import remote from 'remote';
-const app = remote.require('app');
+const {app} = require('electron').remote;
```

Comment:
Also, I think I'm inconsistent with spacing within `{...}`. Anyone have a preference? Pretty sure we can enforce it with eslint.